styles/flex.css

```diff
@@ -0,0 +1,95 @@
+.flex {
+  display: flex;
+}
+
+.flex--inline {
+  display: inline-flex;
+}
+
+/* Flex direction */
+
+.flex__dir--col {
+  flex-direction: column;
+}
+
+.flex__dir--row {
+  flex-direction: row;
+}
+
+.flex__dir--col-rev {
+  flex-direction: column-reverse;
+}
+
+.flex__dir--row-rev {
+  flex-direction: row-reverse;
+}
+
+/* Flex wrap */
+
+.flex__wrap--wrap {
+  flex-wrap: wrap;
+}
+
+.flex__wrap--no {
+  flex-wrap: no-wrap;
+}
+
+.flex__wrap--wrap-rev {
```

Comment:
Better to use 3 words (as less as possible) for utility classes, as it will be easier to use in class names, for eg. flex__wrap-rev is enough here.

Reply:
I was also thinking about that but my brain wasn't able to shorten it that time haha. I will do it.